joomla/joomla-cms
# [4.0] Remove button Cancel from Registration form

For A11Y reasons, as reported by @zwiastunsw and because in Web browser situations, like this, a 'Cancel' button is not needed, as the user can simply navigate away, if they don't want to commit, the 'Cancel' button is removed. 

In any case, there is no 'cancel' type for the button element, in HTML, so this can only be done by using CSS to make a <a> element look like a button or by making a normal button and adding Javascript. Neither is a good solution for an unnecessary button.

Therefore, it is being removed.

### Summary of Changes

'Cancel' button is removed.

### Testing Instructions

Open the registration form and see that the 'Cancel' button no longer exists and that the screenreader behaves correctly.

### Expected result

Better screenreader behaviour.

### Actual result



### Documentation Changes Required
